### PR TITLE
buildah bud should require a context directory or URL

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
Fixes https://github.com/projectatomic/buildah/issues/703

**Comment by rhatdan** [comment]
Fixes https://github.com/projectatomic/buildah/issues/504

**Comment by TomSweeneyRedHat** [comment]
@rhatdan looks like some bud tests need some adjusting

**Comment by rhatdan** [comment]
@nalind @TomSweeneyRedHat @umohnani8 Finally green buttons. PTAL
We now match the way docker works.  IE You always need to specify context.  This patch also fixes other breakage in the testing system.

**Comment by rhatdan** [comment]
@nalind @TomSweeneyRedHat @umohnani8 PTAL

**Comment by umohnani8** [comment]
LGTM

**Comment by TomSweeneyRedHat** [comment]
Sorry @rhatdan thought I LGTM'd this already.
LGTM

File: tests/validate/git-validation.sh [hunk 1]
```diff
@@ -8,6 +8,6 @@ if ! which git-validation > /dev/null 2> /dev/null ; then
 fi
 if test "$TRAVIS" != true ; then
 	#GITVALIDATE_EPOCH=":/git-validation epoch"
-	GITVALIDATE_EPOCH="bf40000e72b351067ebae7b77d212a200f9ce051"
+	GITVALIDATE_EPOCH="25f4e8ec639044bff4ab393188d083782f07b61c"
```
Comment (nalind) [inline tests/validate/git-validation.sh]: What are we fixing by changing this?

**Comment by nalind** [comment]
What's the issue that the `golang.org/x/sys/unix` update fixes?  Can we include a mention of it in the log message?

**Comment by rhatdan** [comment]
@nalind Updated the log.

**Comment by rhatdan** [comment]
@rh-atomic-bot r=TomSweeneyRedHat

**Pull request closed without merging**